src/components/select/select-textbox/select-textbox.component.tsx

```diff
@@ -234,12 +216,7 @@ const SelectTextbox = React.forwardRef(
     function renderSelectText() {
       return (
```

Comment:
**nitpick (non-blocking):** same comment from above might apply here (if its correct in the first place)

Reply:
it's a much fairer point for this one that the one above. In fact since this is only used in one place I can probably remove the function entirely and just inline the JSX at the point it's currently called. Happy to do that 👍